examples/freesolv/README.md

```diff
@@ -0,0 +1,23 @@
+# FreeSolv example
+
+This folder contains an example of optimizing a subset of FreeSolv molecules for hydration free energy
+using Reversible Jump MCMC. The `MultiTargetDesigner` that gets instantiated is intended to focus on the states
+whose relative hydration free energy is most favorable.
+
+## Components
+
+### Database
+The database originates from the Mobley Lab [FreeSolv repository](https://github.com/MobleyLab/FreeSolv).
+
+### Preparation
+In this example, we restrict ourselves to molecules that contain a benzene ring. To prepare
+the included database, run `bash prepare_freesolv.sh`. This will generate several output files:
+
+* `filtered_database.smi`: A set of SMILES strings for the molecules to use in the calculation
```

Comment:
@pgrinaway : Don't you want to include the `filtered_database.smi` and maybe the PDF files?

Reply:
Maybe that's better, yeah. Perhaps some people looking it at it will not have an OpenEye license. I'll add those.